src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -42,7 +42,7 @@ public function __construct(Scope $parent = null)
     /**
      * Opens a new child scope.
      *
-     * @return Scope
+     * @return $this
```

Comment:
this should be `@return self` right?
`this` - return exactly the same instance
`self` - return new instance of exactly same type
`static` - return new instance of same type or any child of type